### PR TITLE
completed html-syntax

### PR DESCRIPTION
**Pull request by Ortiz-Andrew**
Screenshot 2023-09-01 at 9 56 42 AM

### CODE REVIEW

File: html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title></title>
```
Comment (shawnkost) [inline html-syntax/index.html]: Exercise name should go here!

File: html-syntax/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  </head>
+  <body>
+    <img
+      src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/384.png"
+      alt="Rayquaza" />
+    <h1>Rayquaza</h1>
+    <p>©Pokemon. ™Nintendo</p>
```
Comment (shawnkost) [inline html-syntax/index.html]: You should be using HTML entities for these characters

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Andrew

**Comment by Ortiz-Andrew** [comment]
Screenshot 2023-09-01 at 10 27 37 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**